test/test_tests.py

```diff
@@ -0,0 +1,360 @@
+import six
+
+from warcio.cli import main
+from warcio.utils import to_native_str
+import warcio.tester
+
+from . import get_test_file
+from .test_cli import patch_stdout
+
+
+def helper(args, expected_exit_value):
+    with patch_stdout() as buff:
+        exit_value = None
+        try:
+            main(args=args)
+        except SystemExit as e:
+            exit_value = e.code
+        finally:
+            assert exit_value == expected_exit_value
+
+        return to_native_str(buff.getvalue())
+
+
+def remove_before_test_data(s):
+    ret = ''
+    for line in s.splitlines(True):
+        if '/test/data/' in line:
+            line = 'test/data/' + line.split('/test/data/', 1)[1]
+        if '\\test\\data\\' in line:
+            line = 'test/data/' + line.split('\\test\\data\\', 1)[1]
+        ret += line
+    return ret
+
+
+def test_torture_missing():
+    files = ['standard-torture-missing.warc']
+    files = [get_test_file(filename) for filename in files]
+
+    args = ['test']
+    args.extend(files)
+
+    expected = """\
```

Comment:
Perhaps it would be a good idea to convert these tests to doctests due to the lengthy values of the expected?

Reply:
I've never seen a lengthy doctest, that seems weird to me. However, it's likely that I'll end up putting these big things into a file, one per warc tested.